src/main/java/mekanism/client/gui/GuiTeleporter.java

```diff
@@ -137,6 +149,11 @@ private void updateButtons() {
     @Override
     public void func_231023_e_() {
         super.func_231023_e_();
+        TeleporterFrequency freq = getFrequency();
+        if (!init && getFrequency() != null) {
+            init = true;
+            privateMode = freq.isPrivate();
+        }
```

Comment:
Is there a reason this is being done, it seems kind of pointless given it already gets set in the constructor.

Reply:
Two actually:
1. its done in the QIO too,
2. It fixes some bug, i don't quite remember which one, or whether something else fixes that too tho.

Reply:
Seems odd but fair enough.